Add standard attention map

Y4NG333 opened the pull request.

Y4NG333 committed:
draw_plot function adds a new input(main)

Y4NG333 committed:
draw_plot function adds a new input(test)

Y4NG333 committed:
Add standard attention map in draw_plot function

Y4NG333 commented:
@albert-no 교수님, 안녕하세요. 저는 draw_plot 함수에 standard attention map을 그리는 기능을 추가했습니다.다음과 같이
![image](https://user-images.githubusercontent.com/91429283/166139870-1c450cab-79b4-442a-bc43-a1c9735d515a.png)
이렇게 standard attention map을 그리면 될까요?

albert-no reviewed (commented):
네 맞습니다. 그림은 좋습니다.
다만 imshow가 아니라 savefig와 같이 이미지를 보여주는대신 저장하기 바랍니다.

Y4NG333 commented:
네,수정하겠습니다.

Y4NG333 committed:
Use savefig to store the attention map

Y4NG333 commented:
@albert-no 교수님, Savefig 함수를 추가했습니다.

albert-no commented:
지금과 같이 그림을 저장하면 파일이름이 동일해서 한번에 여러 파일을 생성할 수 없습니다.
파일이름을 입력으로 받게하고
입력파일이름을 계속 변경해주기 바랍니다.

Y4NG333 committed:
Use filename as function input(main)

Y4NG333 committed:
Use filename as function input(test)

Y4NG333 committed:
Add new input in draw_plot and modified savefig function

Y4NG333 committed:
Add new draw function

Draw and save all the attention maps corresponding to 100 sequences

Y4NG333 committed:
Add a new draw function

it can draw and store the attention map corresponding to all sequences in the batch

Y4NG333 commented:
@albert-no 교수님, 제가 새로운 darw_plot_test 함수를 하나 추가했습니다.
하나의 batch에 있는 모든 attention map을 만들고 저장할 수 있습니다.(100개)
모든 output_attention map과 standard_attention map은 image라는 이름의 폴더에 저장됩니다.

albert-no commented:
@Y4NG333 비슷한 함수를 두개 만드는것은 나중에 문제가 발생할수 있습니다.
두개의 함수를 하나로 합쳐서 모든 경우에 사용할 수 있도록 만들기 바랍니다.

Y4NG333 commented:
네, 알겠습니다. 수정하겠습니다.

Y4NG333 committed:
Removed one draw function and modified the other.

Y4NG333 committed:
Use the new draw function（main）

Y4NG333 committed:
Use the new draw function(test)

Y4NG333 commented:
@albert-no 교수님, 안녕하세요.draw함수 하나를 삭제하고 다른 하나를 수정했습니다.
이 새로운 draw함수 우리의 다양한 drawing 요구사항을 충족시킬 수 있습니다.

albert-no commented:
네 고생많았습니다. merge하기 바랍니다

Y4NG333 commented:
네~
